apisix/plugins/traffic-split.lua

```diff
@@ -238,6 +238,7 @@ function _M.access(conf, ctx)
 
     for _, rule in ipairs(conf.rules) do
         if not rule.match then
+            match_passed = true
```

Comment:
The match_passed  is true by default:
https://github.com/apache/apisix/blob/50d6465fd7b6288f517fd5c408a3171ace7882dd/apisix/plugins/traffic-split.lua#L237

Reply:
But if match_passed is set to false on the first loop, it will remain false instead of true on the second loop
![image](https://user-images.githubusercontent.com/21658981/153547615-8f96d35d-57bd-43e7-a937-04260695c216.png)

Reply:
I see. Can you add a test to cover it?

Reply:
You can refer to https://github.com/apache/apisix/pull/6242#issuecomment-1028625979 and the https://github.com/apache/apisix/blob/master/t/plugin/traffic-split5.t

Reply:
ok, I'll look into it

Reply:
Take a look please